src/en/getting-started.md

```diff
@@ -15,20 +17,26 @@ need:
 To install Juju, you simply need to grab the latest juju-core package from the
 PPA:
 
-    sudo add-apt-repository ppa:juju/stable
-    sudo apt-get update && sudo apt-get install juju-core
+```bash
+sudo add-apt-repository ppa:juju/stable
+sudo apt-get update && sudo apt-get install juju-core
+```
 
-For more information on installing and the current versions available, see [the releases page](reference-releases.html).
+For more information on installing and the current versions available, see
+[the releases page](reference-releases.html).
 
 ## Mac OSX
 
 Juju is in [Homebrew](http://brew.sh/), to install do:
 
-    brew install juju
+```bash
+brew install juju
+```
 
 We also recommend trying Juju in [our Vagrant box](config-vagrant.html).
 
-For more information on installing and the current versions available, see [the releases page](reference-releases.html).
+For more installation information and what versions are available, see [the
```

Comment:
I tend to avoid at pretty much all costs, breaking links across lines - all markdown parsers seem to be particularly bad at  catching these, plus it makes them far harder to spot in the raw text